js/analyzer.js

```diff
@@ -67,9 +67,12 @@ Analyzer.prototype.checkConfig = function() {
  * YoastSEO.Analyzer initialization. Loads defaults and overloads custom settings.
  */
 Analyzer.prototype.init = function( args ) {
+	var metaValues = {
```

Comment:
This object isn't being used anywhere else. Might as well directly inject it into Paper as such:

`args.paper = new Paper( args.keyword, { metaDescription: args.meta } )` or just pass the `args.meta` and let Paper handle the additional wrapping of `args.meta`.